src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -60,7 +62,12 @@ String aboutMe = (String) request.getAttribute("aboutMe");
       </ul>
 
     </div>
-
+    <h2> <%= subject %>'s subscriptions </h2>
+    <% List<UUID> subscriptionsID = (List<UUID>) request.getAttribute("subscriptionsID"); %>
+    <% List<String> conversationNames = (List<String>) request.getAttribute("conversationNames"); %>
+    <% for (int i = 0; i< subscriptionsID.size(); i++) { %>
+    <a class="btn btn-primary" href="/chat/<%=subscriptionsID.get(i)%>" role="button"> <%=conversationNames.get(i)%> </a>
```

Comment:
This hyperlink should go to `/chat/<chat_uuid>`, not subscription id right? If so, we'll need to fetch the conversation ids along with their names, since we're now using their ids in the URL after @vocatwk's change.

Reply:
the subscription_ids are the Ids of the chats

Reply:
Ah, I missed that. Can we rename it to make it easier to understand then? Maybe conversationIds?